vcd/resource_vcd_vapp_vm.go

```diff
@@ -96,13 +107,65 @@ func resourceVcdVAppVm() *schema.Resource {
 				Default:  true,
 			},
 			"network_href": &schema.Schema{
-				Type:     schema.TypeString,
-				Optional: true,
+				Type:       schema.TypeString,
+				Optional:   true,
+				Deprecated: "In favor of networks",
+			},
+			"networks": {
+				ConflictsWith: []string{"ip", "network_name"},
+				ForceNew:      true,
+				Optional:      true,
+				Type:          schema.TypeList,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"orgnetwork": {
+							ForceNew: true,
+							Required: true,
+							Type:     schema.TypeString,
+						},
+						"ip": {
+							Computed:         true,
+							DiffSuppressFunc: suppressIfIPIsOneOf(),
```

Comment:
I'm thinking if we could fix this odd behavior when text attribute is overriden with IP address (like in the old case). What are your thoughts if we only allow to enter IP address here for manual case and return computed in other cases?

Reply:
I would like to go for that. I tried to recreate current behavior.

Reply:
I think as this is new functionality we can make it proper (no one will get a surprise) here (but preserve it in the original implementation until we remove it in future).

Reply:
Added validation for attribute "ip"